plugins/komodo-plugin-service/src/main/java/org/komodo/osgi/PluginService.java

```diff
@@ -294,6 +294,8 @@ private void createDirectory(String path) {
     }
 
     private Bundle findBundleBySymbolicName(String symbolicName) throws Exception {
+        ArgCheck.isNotNull(framework, "framework");
```

Comment:
Seems more appropriate to be an `assert` statement as `framework` is created in the constructor. Same comment for other methods below.